include/seqan3/std/charconv_detail.hpp

```diff
@@ -0,0 +1,843 @@
+// the Software without restriction, including without limitation the rights to
+// use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies
+// of the Software, and to permit persons to whom the Software is furnished to do
+// so, subject to the following conditions:
+
+//     * Redistributions of source code must retain the above copyright notice,
+//       this list of conditions and the following disclaimers.
+
+//     * Redistributions in binary form must reproduce the above copyright notice,
+//       this list of conditions and the following disclaimers in the
+//       documentation and/or other materials provided with the distribution.
+
+//     * Neither the names of the LLVM Team, University of Illinois at
+//       Urbana-Champaign, nor the names of its contributors may be used to
+//       endorse or promote products derived from this Software without specific
+//       prior written permission.
+
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+// FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+// CONTRIBUTORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS WITH THE
+// SOFTWARE.
+
+// =============================================================================
+// Copyrights and Licenses for Third Party Software Distributed with LLVM:
+// =============================================================================
+// The LLVM software contains code written by third parties.  Such software will
+// have its own individual LICENSE.TXT file in the directory in which it appears.
+// This file will describe the copyrights, license, and restrictions which apply
+// to that code.
+
+// The disclaimer of warranty in the University of Illinois Open Source License
+// applies to all code in the LLVM Distribution, and nothing in any of the
+// other licenses gives permission to use the names of the LLVM Team or the
+// University of Illinois to endorse or promote products derived from this
+// Software.
+
+// The following pieces of software have additional or alternate copyrights,
+// licenses, and/or restrictions:
+
+// Program             Directory
+// -------             ---------
+// Autoconf            llvm/autoconf
+//                     llvm/projects/ModuleMaker/autoconf
+//                     llvm/projects/sample/autoconf
+// CellSPU backend     llvm/lib/Target/CellSPU/README.txt
+// Google Test         llvm/utils/unittest/googletest
+// OpenBSD regex       llvm/lib/Support/{reg*, COPYRIGHT.regex}
+// =============================================================================
+
+/*!\file
+ * \brief Copied code from LLVM for std::from_chars and std::to_chars.
```

Comment:
"Provides LLVM's implementation of std::from_chars and std::to_chars for integral types and a custom solution for floating points types."